dot/network/metrics.go

```diff
@@ -0,0 +1,60 @@
+package network
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/ethereum/go-ethereum/metrics"
+)
+
+const (
+	refresh     = time.Second * 10
+	refreshFreq = int64(refresh / time.Second)
+)
+
+// CollectProcessMetrics periodically collects various metrics about the running process.
+func CollectProcessMetrics() {
```

Comment:
i think since these metrics aren't specific to the network package, they could probably go in a new package for `metrics` instead

Reply:
Done